### PR TITLE
integrated anonymous session in client for specific route

### PR DESCRIPTION
**Pull request by TechAlchmy**
integrated anonymous session in client for specific route

### CODE REVIEW

**Review by lukeocodes — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@uidotdev/usehooks": "^2.4.1",
     "ai": "^2.2.33",
     "classnames": "^2.5.1",
+    "framer-motion": "^11.3.31",
```
Comment (lukeocodes) [inline package.json:24]: This seems superfluous

**Review by lukeocodes — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jpvajda) [reply package.json:24]: Is this comment saying we don't want to use this library?

Comment (TechAlchmy) [reply package.json:24]: I used this when I remove unnecessary  code and clean this project that is is requiems. if this package is not install than issue is not fixed So I use this

Comment (lukeocodes) [reply package.json:24]: `framer-motion` is an animation library. I don't understand why it is installed when you're not using it anywhere

**Pull request closed without merging**